examples/lighting-app/linux/BUILD.gn

```diff
@@ -19,6 +19,12 @@ import("${chip_root}/src/app/common_flags.gni")
 
 assert(chip_build_tools)
 
+import("${chip_root}/examples/common/pigweed/pigweed_rpcs.gni")
+
+if (chip_enable_pw_rpc) {
+  import("//build_overrides/pigweed.gni")
```

Comment:
nit: move below into the 2nd if (chip_enable_pw_rpc)

Reply:
Done